templates/account/user_pref_metadata.html

```diff
@@ -6,7 +6,9 @@
             <p><strong>Language(s):</strong> {{form.data.languages|join(', ') }}</p>
         {% endif %}
         {% if form.data.locations %}
-            <p><strong>Location(s):</strong> {{form.data.locations|join(', ') }}</p>
+            <p><strong>Location(s):</strong>
+                {% for location in form.data.locations %} {{ '(' ~ location ~ ')' if location|length == 2 else location }}{% if not loop.last %}, {% endif %} {% endfor %}
```

Comment:
Could we adjust this to display the format:

Country Name One (CO), Country Name Two (CP), Country Name Three (CQ)

```python
<p><strong>Location(s):</strong>  
{% for i in range(0, form.data.locations|length, 2) %}  
{{ form.data.locations[i] }} ({{ form.data.locations[i+1] }}){% if not loop.last %}, {% endif %}  
{% endfor %}  
</p>  
```